README.md

```diff
@@ -21,6 +21,10 @@ Please refer to the documentation for more information: https://mapswipe-workers
 - MapSwipe OSM-Wiki: https://wiki.openstreetmap.org/wiki/MapSwipe
 
 
+## Setup 
+
+see https://mapswipe-workers.readthedocs.io/en/master/dev_setup.html
+
```

Comment:
Is this necessary? Above, the documentation is already linked:

If you want to keep it, I suggest:

```suggestion
## Development Setup 

Please see here: https://mapswipe-workers.readthedocs.io/en/master/dev_setup.html

```